mobile/lib/shared/services/network.service.dart

```diff
@@ -4,15 +4,22 @@ import 'dart:convert';
 import 'package:dio/dio.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:hive/hive.dart';
+import 'package:http_parser/http_parser.dart';
+import 'package:image_picker/image_picker.dart';
 import 'package:immich_mobile/constants/hive_box.dart';
 import 'package:immich_mobile/utils/dio_http_interceptor.dart';
+import 'package:immich_mobile/utils/files_helper.dart';
 
 class NetworkService {
+  late final Dio dio;
+
+  NetworkService() {
+    dio = Dio();

```

Comment:
This is very cool, to instantiate only once and add the interceptor at the constructor. Love this change!